src/deluge/dsp/filter/svf.cpp

```diff
@@ -86,14 +110,21 @@ inline q31_t SVFilter::doSVF(int32_t input, SVFState& state) {
 	high = high - 2 * multiply_32x32_rshift32(band, q);
 	band = 2 * multiply_32x32_rshift32(high, fc) + band;
 
+	//notch = high + low;
+	lowi = lowi + low;
+	highi = highi + high;
+	bandi = bandi + band;
+
+	q31_t result = multiply_32x32_rshift32_rounded(lowi, c_low);
+	result = multiply_accumulate_32x32_rshift32_rounded(result, highi, c_high);
+	if (band_mode) {
+		result = multiply_accumulate_32x32_rshift32_rounded(result, bandi, c_band);
+	}
+
 	//saturate band feedback
 	band = getTanHUnknown(band, 3);
-	//notch = high + low;
-	q31_t result = multiply_32x32_rshift32_rounded(lowi + low, c_low);
-	result = multiply_accumulate_32x32_rshift32_rounded(result, highi + high, c_high);
-	result = multiply_accumulate_32x32_rshift32_rounded(result, bandi + band, c_band);
 	//result = multiply_accumulate_32x32_rshift32_rounded(0, notchi+notch, c_notch);
-	result = 2 * result; //compensate for division by two on each multiply
+	result = 3 * result; //compensate for division by two on each multiply
```

Comment:
Is the comment here stale now?

Reply:
Sort of, 2 is to compensate for the division on the multiply. I bumped it to 3 to match the volume of the ladder filters since it's easier than doing a *1.5 later 

I'll update the comment